### ISSUE TITLE
Log inside windows service

### ISSUE THREAD

**Issue by russocorp** [danieleteti/loggerpro#9]
Hi, this is not exactly a issue, i think i'm do not know how use correctly the LoggerPro.
I have a windows service to update database and exe on my clients and i'm using the file "LoggerProConfig" to write logs and see errors or other important messages.
But, when i use for example "Log.Error('TEST', 'TEST');" i don´t see new line in file.
There is anything i need to do to correct this problem?
Thank you since now.

**Comment by danieleteti** [comment]
Can you show your ```LoggerProConfig.pas``` file?

**Comment by russocorp** [comment]
Good morning. 
I copied the file from Samples/02_file_appender and make change with folder creation, to save file in [exepath]\Log.
This is the file i´m using. I changed extension to txt to upload in github.


[LoggerProConfig.txt](https://github.com/danieleteti/loggerpro/files/854933/LoggerProConfig.txt)

**Comment by danieleteti** [comment]
Did you solved the problem?

**Comment by russocorp** [comment]
No. I'll try later and say if I found a solution. New tax rules coming in my country, I ended up developing another project.
